src/map/map.cpp

```diff
@@ -476,7 +474,20 @@ void display_data::render(glm::vec2 screen_size, glm::vec2 offset, float zoom, m
 				count.push_back(border.count);
 			}
 		}
-
+		glMultiDrawArrays(GL_TRIANGLES, &first[0], &count[0], GLsizei(count.size()));
+	}
+	if(zoom <= 5) { // Coastal borders
```

Comment:
this will draw more than coasts. It will draw all the inland impassible borders as well